app/models/text_field_type.rb

```diff
@@ -37,7 +37,7 @@ def text_length
 
   def text_unique
     unless field.field_items.jsonb_contains(:data, text: text).empty?
-      errors.add(:text, "#{field.name} Must be unique")
+      errors.add(:text, "#{field.name} Must be unique") if field_item.content_item_id == nil
```

Comment:
@MKwenhua This still will not work - this just disables this validation if they're editing the `ContentItem`, which isn't appropriate. We still need to validate for both Create and Update.
Instead of passing the `field_item`, let's use `existing_data`, which is already passed to `FieldType` - with this, we can see what the value of `text` was previously. If it matches the text we're currently validating against, we can skip adding an error.
Additionally, we already have a conditional we can use on the previous line - no need to add it inline here.